### PR TITLE
Move to stdlib and extend the models for `fmt` package

### PR DESCRIPTION
**Pull request by gagliardetto**
`codebox` command:

```bash
codebox --out-dir=./generated/latest --pkg=/usr/local/go/src/fmt --http
```
---

Part of #167

### CODE REVIEW

**Review by smowton — APPROVED** [review]

File: ql/src/semmle/go/frameworks/stdlib/Fmt.qll [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+      hasQualifiedName("fmt", "Sscanln") and
+      (inp.isParameter(0) and outp.isParameter(any(int i | i >= 1)))
+    }
+
+    override predicate hasTaintFlow(FunctionInput input, FunctionOutput output) {
+      input = inp and output = outp
+    }
+  }
+
+  /** The `Scan` function or one of its variants, all of which read from os.Stdin */
```
Comment (max-schaefer) [inline ql/src/semmle/go/frameworks/stdlib/Fmt.qll:89]:
```suggestion
  /** The `Scan` function or one of its variants, all of which read from `os.Stdin`. */
```

File: ql/src/semmle/go/frameworks/stdlib/Fmt.qll [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+  class Scanner extends Function {
+    Scanner() { this.hasQualifiedName("fmt", ["Scan", "Scanf", "Scanln"]) }
+  }
+
+  /**
+   * The `Fscan` function or one of its variants,
+   * all of which read from a specified io.Reader
```
Comment (max-schaefer) [inline ql/src/semmle/go/frameworks/stdlib/Fmt.qll:96]:
```suggestion
   * all of which read from a specified `io.Reader`.
```

File: ql/src/semmle/go/frameworks/stdlib/Fmt.qll [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+   */
+  class FScanner extends Function {
+    FScanner() { this.hasQualifiedName("fmt", ["Fscan", "Fscanf", "Fscanln"]) }
+
+    /**
+     * Returns the node corresponding to the io.Reader
```
Comment (max-schaefer) [inline ql/src/semmle/go/frameworks/stdlib/Fmt.qll:102]:
```suggestion
     * Gets the node corresponding to the `io.Reader`
```

**Review by max-schaefer — CHANGES_REQUESTED** [review]
LGTM, but I'd like to tack on a few small consistency improvements if I may.

**Comment by max-schaefer** [comment]
image

Nice SHA you managed to hit there!

**Review by max-schaefer — APPROVED** [review]

**Comment by gagliardetto** [comment]
> image
> 
> Nice SHA you managed to hit there!

Nice! :sparkles:

**Pull request merged**